client/app/telecom/pack/xdsl/modem/connectedDevices/pack-xdsl-modem-connectedDevices.controller.js

```diff
@@ -80,8 +80,6 @@ angular.module('managerApp').controller('XdslModemConnectedDevicesCtrl', functio
     }).catch((err) => {
       TucToast.error($translate.instant('xdsl_modem_refresh_error'));
       return $q.reject(err);
-    }).finally(() => {
-      self.loading = false;
```

Comment:
It will keep loading if you remove that I suppose

Reply:
It's the goal.